ironfish/src/wallet/walletdb/accountValue.test.ts

```diff
@@ -22,6 +22,32 @@ describe('AccountValueEncoding', () => {
         hash: Buffer.alloc(32, 0),
         sequence: 1,
       },
+      multiSigKeys: undefined,
+    }
+    const buffer = encoder.serialize(value)
+    const deserializedValue = encoder.deserialize(buffer)
+    expect(deserializedValue).toEqual(value)
+  })
+
+  it('serializes an object with multiSigKeys into a buffer and deserializes to the original object', () => {
+    const encoder = new AccountValueEncoding()
+
+    const key = generateKey()
+    const value: AccountValue = {
+      id: 'id',
+      name: 'foobar👁️🏃🐟',
+      incomingViewKey: key.incomingViewKey,
+      outgoingViewKey: key.outgoingViewKey,
+      publicAddress: key.publicAddress,
+      spendingKey: key.spendingKey,
```

Comment:
I thought we decided we would have an additional field in `multiSigKeys` for `groupSpendingKey` and make this null? That way these accounts will be mostly treated as view only accounts.

Reply:
this spending key should always be null if `multiSigKeys` is defined. i'll change that for the test and look into whether there's a straightforward way to formalize it

i think it's simpler to store the `proofGenerationKey` in the multisig keys instead of the group spending key since we'd only need that key to derive the proof generation key

Reply:
Oh yea that makes sense, and outgoing view key will be stored in the main account structure. SGTM